examples/using-guidebot-flags.md

```diff
@@ -0,0 +1,106 @@
+  # Using Switch Cases with Guidebot
```

Comment:
You indented the entire file with 2 spaces.

Reply:
How the actual fuck.